yksmp/src/lib.rs

```diff
@@ -201,8 +201,8 @@ impl StackMapParser<'_> {
 
             let location = match kind {
                 0x01 => {
-                    self.read_i32();
-                    Location::Register(dwreg, size)
+                    let offset = self.read_i32();
```

Comment:
not sure that the variable is much use here: just `Location::Register(dwreg, size, self.read_i32());` ?

Reply:
Semantically no, but it gives context about what value we are extracting.

Reply:
OK, I don't feel strongly about this.